lib/v2/screens/profile_screens/guardians/invite_guardians/interactor/mappers/invite_guardian_state_mapper.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/v2/screens/profile_screens/guardians/invite_guardians/interactor/viewmodel/invite_guardians_page_commands.dart';
+import 'package:seeds/v2/screens/profile_screens/guardians/invite_guardians/interactor/viewmodel/invite_guardians_state.dart';
+
+class InviteGuardiansStateMapper extends StateMapper {
+  InviteGuardiansState mapResultToState(InviteGuardiansState currentState, Result result) {
+    if (result.isError) {
+      return currentState.copyWith(
+          pageState: PageState.failure,
+          pageCommand: ShowErrorMessage('Oops, something went wrong. Please try again later'));
```

Comment:
Isn't it easier to have the text already in the UI? So the i18n file is not imported here.
Also I think this text is too long to fit in a snack.

Reply:
i guess we have not discussed best practices with i18 files and mappers or blocs 
But i think we should be sending the text down to the UI from the state or the command and not getting it in the UI. 

For example. 

`The sent 1 Peso to Pepe`
`The user sent 2 Pesos to Pepe`

Here we have to deal with plurals. To accomplish this we need to do something like 
if(sentAmount == 1) {
`The sent 1 Peso to Pepe`
} else {
`The user sent 2 Pesos to Pepe`
}

I would prefer to have this logic in the mapper and not in the UI. Which means we need to have strings in the mappers and not in the UI.

Reply:
I agree.
If in fact in Invite I create and handle the alert texts from the mapper, it happens that I also have made screens only show one error and that is why I left the text in the UI.